docs/network/architecture.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Etherlink architecture
+---
+
+Etherlink's main components are its nodes and the sequencer.
+
+## Sequencer
+
+Etherlink relies on a sequencer to process transactions.
+It receives transactions from nodes and packages them into _blueprints_, which are like blocks.
+In the context of Etherlink, the transactions in a blueprint are final, but they are not final for layer 1 until they are published on layer 1.
```

Comment:
I don’t agree with this, this is not nuanced enough. It is final _if you trust the sequencer_